xcode/DirectBindingsApp/DirectBindingsApp/app-batteries/BlockchainObserver.swift

```diff
@@ -478,6 +478,18 @@ class BlockchainObserver {
 		return result
 	}
 
+    public func isMonitoring() async -> Bool {
+        return await self.monitoringTracker.startTracking()
```

Comment:
that seems like a very dangerous method if it's called before the monitor is running, seeing how it will prevent it from ever being started.

Reply:
will this be better?

```
    public func isMonitoring() async throws -> Bool {
        try await preloadMonitor()
        return await self.monitoringTracker.startTracking()
    }
```

Reply:
nah, just add a method to the monitoring Tracker to indicate whether or not it's tracking

Reply:
it will be reliable at the moment simply because it's an actor